src/libs/extra/interactive.liq

```diff
@@ -495,7 +495,7 @@ def interactive.harbor(
           }
         }
         if (interactive[i].type != 'button') {
-          data = data.concat(interactive[i].name+'='+interactive[i].value)+'&';
+          data = data.concat(interactive[i].name+'='+encodeURIComponent(interactive[i].value))+'&';
```

Comment:
Is it just me, or `encodeURIComponent` doesn't exist?

Reply:
`encodeURIComponent` is on javascript side. It's standard

Reply:
Just figured out this is a JS code.